examples/syntax.nit

```diff
@@ -0,0 +1,182 @@
+#! /usr/bin/env nitc
+
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Test case for syntax highlighters.
+#
+# Also test spell-checking and comment recognition.
+#
+# SEE: [Nit](http://nitlanguage.org)
+# XXX: Dummy task
+module syntax is
+	new_annotation new_annotation
+
+	conditional
+
+	deprecated
+	fixed
+	lazy
+	noinit
+	readonly
+	writable
+	optional
+	autoinit
+	noautoinit
+	lateinit
+	nosuper
+	old_style_init
+	abstract
+	intern
+	extern
+	no_warning("fictive-module")
+	generated
+
+	pkgconfig("a libray from outer space")
```

Comment:
I was unsure since I don’t remember seeing a module that mix more than one FFI language. Since the module can be made valid, I think I should simply add a to-do for the placement and use of the annotations. Note that I have to use all the annotations in order to ensure that they are all highlighted.

I placed the module in `examples` because it’s more a demo than a test since, for now at least, we have to manually/visually inspect the rendering to determine if the language definitions are right or not.